src/DoctrineMongoODMModule/Options/Configuration.php

```diff
@@ -162,8 +197,8 @@ public function getDriver()
 
     /**
      *
-     * @param boolean|int $generateProxies
-     * @return \DoctrineMongoODMModule\Options\Configuration
+     * @param boolean $generateProxies
```

Comment:
This should be `int` and for BC there could be something like in bundle: https://github.com/doctrine/DoctrineMongoDBBundle/blob/ee1cdebc02d4ed49e90e9ad15737b1be03ac3eee/DependencyInjection/Configuration.php#L54-L58